src/patternfly/components/Card/card.scss

```diff
@@ -213,6 +223,11 @@
   + .pf-c-card__footer {
     padding: 0;
   }
+
+  .pf-c-dropdown {
```

Comment:
I think moving the margin that offsets the alignment from the actions container to an individual component is going to cause inconsistencies. You could use other plain things here, too - button, options menu, etc, and then the alignment would be different.

As an example, this is what it looks like currently if you use a dropdown and a button

<img width="852" alt="Screen Shot 2021-05-05 at 10 32 28 AM" src="https://user-images.githubusercontent.com/35148959/117168601-eb9f1f00-ad8d-11eb-95ca-79a15b16d51d.png">

And here is what that looks like with this change3

<img width="871" alt="Screen Shot 2021-05-05 at 10 32 20 AM" src="https://user-images.githubusercontent.com/35148959/117168574-e510a780-ad8d-11eb-9cf5-9bfbdcaa4b6b.png">